### DISCUSSION TITLE
Playcover 2.02 option key not responsive, cannot summon the mouse nor hide it

### DISCUSSION THREAD

**Discussion by Victor132725** [PlayCover/PlayCover#642] [Q&A]
Im currently using play cover 2.02 on my mac. The operating system is macOS 13.01. Sometimes if I restart the computer it works for a while, but the option key alway end up not working. Recently this method also stopped working. It works for around 30 seconds and then the option key becomes unresponsive and my mapped keys also stop working. To this point I can't do anything other than force quitting it. 

Methods I used to try to fix it:
1. restart the computer
2. restart playcover
3. restart the side-loaded application
4. reinstalling playcover
5. Updating to the newest version of playcover
6. updating to the newest ipa (using the newest possible ipa file)
7. reinstalling the ipa file

BTW the application I'm using is Genshin Impact

**Comment by amirsaam** [comment]
Keymapping freeze is caused by an Apple bug that has been fixed in macOS 13.1, we can't do anything about it but recommend users to update to macOS 13.1
